apps/src/codebridge/FileTabs/FileTab.tsx

```diff
@@ -17,30 +20,31 @@ const FileTab = ({file}: FileTabProps) => {
   const activeFile = getActiveFileForProject(project);
   const {iconName, iconStyle, isBrand} = getFileIconNameAndStyle(file);
   const iconClassName = isBrand ? 'fa-brands' : undefined;
+  const isActive = file.active || file === activeFile;
+  const className = isActive
+    ? classNames(moduleStyles.fileTab, moduleStyles.active)
+    : moduleStyles.fileTab;
```

Comment:
`classNames` accepts an object which can toggle a class on/off with a boolean value. This should be re-written as:

```
const className = classNames(moduleStyles.fileTab, { [moduleStyles.active] : isActive } );
```

That way it always adds on the `fileTab` class (non-redundantly) and only the `active` class if `isActive` is true.